builder/slide_and_fold_storage.cc

```diff
@@ -175,11 +175,17 @@ class slide_and_fold : public node_mutator {
     // Overlap between iteration i and i + 1.
     expr overlap;
 
-    // Which loop this fold is for.
+    // Depths of the loop this fold is for.
     std::size_t loop;
```

Comment:
Is this actually still used? I think I added this for the check this PR is replacing?

Reply:
It was also used for somethin else, but should be fixed now.